building/build-debs/homeworld-keysystem/src/keygen/generate.go

```diff
@@ -24,7 +24,7 @@ func GenerateTLSSelfSignedCert(key *rsa.PrivateKey, name string, present_as []st
 
 	certTemplate := &x509.Certificate{
 		KeyUsage:    x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
-		ExtKeyUsage: []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
+		ExtKeyUsage: []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth},
```

Comment:
To what is this certificate used to authenticate as a client?

Reply:
AHAHAHAHAHAHA

it's not. but things that it *issues* need to authenticate as clients, which apparently means this needs to be here.